docs/usage.md

```diff
@@ -199,13 +199,19 @@ NXF_OPTS='-Xms1g -Xmx4g'
 
 Stable releases will be becoming more infrequent as Nextflow shifts its development model to becoming more dynamic via the usage of plugins. This will allow functionality to be added as an extension to the core codebase with a release cycle that could potentially be independent to that of Nextflow itself. As a result of the reduction in stable releases, some pipelines may be required to use Nextflow `edge` releases in order to be able to exploit cutting "edge" features e.g. version 3.0 of the nf-core/rnaseq pipeline requires Nextflow `>=20.11.0-edge` in order to be able to directly download Singularity containers over `http` (see [nf-core/rnaseq#496](https://github.com/nf-core/rnaseq/issues/496)).
 
-There are a number of ways you can install Nextflow `edge` releases, the main difference with stable releases being that you have to `export` the version you would like to install before issuing the appropriate installation commands as highlighted below.
+There are a number of ways you can install Nextflow `edge` releases, the main difference with stable releases being that you have to `export` the version you would like to install before issuing the appropriate installation/execution commands as highlighted below.
 
-* If you have Nextflow installed already you can update it using the commands below:
+* If you have Nextflow installed already, you can issue the version you would like to use on the same line as the pipeline command and it will be fetched if required before the pipeline execution.
+
+```bash
+NXF_VER="20.11.0-edge" nextflow run nf-core/rnaseq -profile test,docker -r 3.0
+```
+
+* If you have Nextflow installed already, another alternative to the option above is to `export` it as an environment variable before you run the pipeline command:
 
 ```bash
 export NXF_VER="20.11.0-edge"
-nextflow self-update
+nextflow run nf-core/rnaseq -profile test,docker -r 3.0
```

Comment:
`nf-core/rnaseq` &rarr; `nf-core/viralrecon`

Reply:
This is just an example as used in the preceeding text that explains why we may need to use edge versions.